### PR TITLE
Remove outdated Application Insights docs

### PR DESCRIPTION
**Pull request by trask**

### CODE REVIEW

File: docs-ref-services/latest/appinsights.md [hunk 1]
```diff
@@ -13,42 +13,6 @@ ms.devlang: java
...
-    telemetryClient.trackMetric(sample);
-```
-
-> [!div class="nextstepaction"]
-> [Explore the Client APIs](/java/api/overview/azure/appinsights/client)
-
-## Samples
-
-Explore more [sample Java code for Application Insights](https://azure.microsoft.com/en-us/resources/samples/?term=insights&platform=java) you can use in your apps.
+Detect, triage, and diagnose issues in your apps and services with [Application Insights Java](/azure/azure-monitor/app/java-in-process-agent).
```
Comment (scottaddie) [inline docs-ref-services/latest/appinsights.md:18]:
```suggestion
Detect, triage, and diagnose issues in your apps and services with [Application Insights for Java](/azure/azure-monitor/app/java-in-process-agent).
```

File: docs-ref-services/legacy/appinsights.md [hunk 2]
```diff
@@ -13,42 +13,6 @@ ms.devlang: java
...
-    telemetryClient.trackMetric(sample);
-```
-
-> [!div class="nextstepaction"]
-> [Explore the Client APIs](/java/api/overview/azure/appinsights/client)
-
-## Samples
-
-Explore more [sample Java code for Application Insights](https://azure.microsoft.com/en-us/resources/samples/?term=insights&platform=java) you can use in your apps.
+Detect, triage, and diagnose issues in your apps and services with [Application Insights Java](/azure/azure-monitor/app/java-in-process-agent).
```
Comment (scottaddie) [inline docs-ref-services/legacy/appinsights.md:18]:
```suggestion
Detect, triage, and diagnose issues in your apps and services with [Application Insights for Java](/azure/azure-monitor/app/java-in-process-agent).
```

File: docs-ref-services/preview/appinsights.md [hunk 3]
```diff
@@ -13,42 +13,6 @@ ms.devlang: java
...
-    telemetryClient.trackMetric(sample);
-```
-
-> [!div class="nextstepaction"]
-> [Explore the Client APIs](/java/api/overview/azure/appinsights/client)
-
-## Samples
-
-Explore more [sample Java code for Application Insights](https://azure.microsoft.com/en-us/resources/samples/?term=insights&platform=java) you can use in your apps.
+Detect, triage, and diagnose issues in your apps and services with [Application Insights Java](/azure/azure-monitor/app/java-in-process-agent).
```
Comment (scottaddie) [inline docs-ref-services/preview/appinsights.md:18]:
```suggestion
Detect, triage, and diagnose issues in your apps and services with [Application Insights for Java](/azure/azure-monitor/app/java-in-process-agent).
```

**Comment by scottaddie** [comment]
@danieljurek Is it possible to put a redirect in place? I'm wondering if a better solution would be to redirect from the 3 modified docs to /azure/azure-monitor/app/java-in-process-agent. Thoughts?

**Comment by danieljurek** [comment]
@scottaddie brings up a really good point. A redirection is probably the correct thing to do here since the text itself is redirecting the user and adding a click. 

Here's how to create a redirect: https://review.docs.microsoft.com/en-us/help/contribute/redirect-articles?branch=master#master-redirect-file

I have a couple more questions to clear up w/ the docs team before we do this: 

* Can we delete the markdown files if there's a redirect? (we've had problems in the past with redirects and markdown files which weren't published) 
* If we can't delete the markdown files is there a better way to do redirects (put redirect info in a separate repo, etc.)? Bespoke configurations in docs repos make the docs publishing process more fragile... we've hit downtime because a redirect deletion snuck into our configs in a large PR before.

**Comment by trask** [comment]
hi all! is there anything further needed from me on this PR? thx

**Comment by scottaddie** [comment]
@danieljurek We spoke briefly about this before the holiday break. Any updates to provide?

**Comment by danieljurek** [comment]
Yeah. Thanks for the patience. I'll open a PR to do some redirects today :)

**Comment by scottaddie** [comment]
Closing, as Daniel's PR replaces this one.

**Pull request closed without merging**